acceptance/localcluster/localcluster.go

```diff
@@ -217,7 +217,17 @@ func (l *Cluster) createVolumes() {
 		panic(err)
 	}
 
-	l.CertsDir, err = ioutil.TempDir(os.TempDir(), "localcluster.")
+	// Create the temporary certs directory in the current working
+	// directory. Boot2docker's handling of binding local directories
+	// into the container is very confusing. If the directory being
+	// bound has a parent directory that exists in the boot2docker VM
```

Comment:
I think it's simpler than that: When boot2docker creates the VM, it mounts /Users/ (i.e. the mac version of /home/) as a virtualbox shared folder. Directories under /Users will work but nothing else will.